src/UI/Buyer/scripts/inject-css.js

```diff
@@ -25,9 +28,50 @@ const hashedCssFilename = cssManifestJson[originalCssFilename]
 if (!hashedCssFilename) {
   throw new Error(`Unable to find css file for ${appID}`)
 }
-const updatedIndexHtml = indexHtml.replace(
+let updatedIndexHtml = indexHtml.replace(
   '<!-- client-theme-placeholder -->',
   `<link href="${hashedCssFilename}" rel="stylesheet" />`
 )
 
+/**
+ * Inject conditional script (Sitecore CDP) into html
+ */
+
+const appConfig = fs.readFileSync(
+  `${sourceDir}/assets/appConfigs/${appID}.json`,
+  'utf-8'
+)
+const clientKey = appConfig.sitecoreCDPApiClient
```

Comment:
Should there be a check against appConfig.useSitecoreCDP here before adding inject-css.js?